app/mailers/user_mailer.rb

```diff
@@ -9,23 +9,30 @@ class UserMailer < ActionMailer::Base
   end
 
   # checks the status of the current reservation and sends the appropriate email
-  # receipt forces a check out receipt to be sent
-  def reservation_status_update(reservation, receipt = false) # rubocop:disable all
-    set_app_config
-
-    @reservation = reservation
-    @status = @reservation.human_status
+  # force overrides and sends the specified email
 
-    return if !receipt && @status == 'returned' && @reservation.overdue &&
-              @reservation.equipment_model.late_fee == 0
+  VALID_STATUSES = ['checked out', 'denied', 'due today', 'missed', 'overdue',
```

Comment:
Sorry to nitpick, but let's replace this with a [`%w()` literal](http://ruby-doc.org/core-2.2.0/doc/syntax/literals_rdoc.html#label-Percent+Strings) instead; I believe that's the preference in the [style guide](https://github.com/bbatsov/ruby-style-guide#percent-w).